lib/presentation/widgets/not_have_account_widget.dart

```diff
@@ -17,14 +20,14 @@ class NotHaveAccount extends StatelessWidget {
       child: Column(
         children: [
           InkWell(
-            child: const Text.rich(
+            child: Text.rich(
               TextSpan(
-                text: notHaveAccount,
+                text: notHaveAccount.tr(),
                 style: CustomTheme.notHaveAccountProperty,
                 children: [
                   spaceTextSpan,
                   TextSpan(
-                    text: singUp, // TODO: refactored
+                    text: singUp.tr(), // TODO: refactored
```

Comment:
why to use static constant for this and not use LocaleKeys.sing_in directly in a build? and remove todo